packages/accordion/src/index.js

```diff
@@ -0,0 +1,480 @@
+////////////////////////////////////////////////////////////////////////////////
+// Welcome to @reach/accordion!
+
+import React, {
+  cloneElement,
+  createContext,
+  forwardRef,
+  useContext,
+  useEffect,
+  useMemo,
+  useRef,
+  useState
+} from "react";
+import { node, func, number, string, bool } from "prop-types";
+import warning from "warning";
+import { wrapEvent, useUpdateEffect } from "@reach/utils";
+import { useId } from "@reach/auto-id";
+
+const { isArray } = Array;
+
+// A11y reference:
+//   - https://www.w3.org/TR/wai-aria-practices/examples/accordion/accordion.html
+//   - https://www.scottohara.me/blog/2018/09/03/details-and-summary.html
+//   - https://inclusive-components.design/collapsible-sections/
+
+// TODO: Screen reader testing
+
+const AccordionContext = createContext({});
+const useAccordionContext = () => useContext(AccordionContext);
+
+////////////////////////////////////////////////////////////////////////////////
+export const Accordion = forwardRef(function Accordion(
+  {
+    allowMultiple: _allowMultiple = true,
+    allowToggle: _allowToggle = true,
+    as: Comp = "div",
+    children,
+    index: controlledIndex = undefined,
+    itemElement = "details",
```

Comment:
I wouldn't recommend defaulting to `details` as [it's breaking in Edge](https://caniuse.com/#feat=details).